[core]/es_extended/server/main.lua

```diff
@@ -1,6 +1,7 @@
 SetMapName('San Andreas')
 SetGameType('ESX Legacy')
 
+local OnesyncState = GetConvar('onesync', 'off')
```

Comment:
I think you should name is oneSyncState.

Reply:
👍🏼